source/blog/2015-10-27-cappcon-2016.markdown

```diff
@@ -0,0 +1,23 @@
+---
+title: CappCon 2016
+author: Alexandre Wilhelm
+author_email: alexandre.wilhelmfr@gmail.com
+date: '2015-10-27'
+categories:
+- Cappuccino
+- Events
+- Conferences
+tags: []
+---
+
+Hey everybody,
+
+The Cappuccino Core Team are planning to organize a conferance about Cappuccino and how they are using it.
```

Comment:
The Cappuccino Core Team is trying to organize an event around Cappuccino next year.

> and how they are using it

Not sure this is necessary